public/static/docs/get-started/add-files.md

```diff
@@ -10,10 +10,10 @@ $ dvc get https://github.com/iterative/dataset-registry \
           get-started/data.xml -o data/data.xml
 ```
 
-> `dvc get` can use any <abbr>DVC project</abbr> hosted on a Git repository to
-> find the appropriate [remote storage](/doc/command-reference/remote) and
-> download <abbr>data artifacts</abbr> from it. (It works like `wget`, but for
-> DVC repositories.) In this case we use
+> `dvc get` can use any Git repository to find the appropriate
+> [remote storage](/doc/command-reference/remote) and download <abbr>data
```

Comment:
so this whole note makes sense for DVC projects (only DVC projects have storage remotes)

Reply:
I see. I'll revert this one.

Reply:
Yep. The context here is specific to DVC projects. Please revert the change @fabiosantoscode. Thanks

Reply:
Reverted, thanks!